### PR TITLE
Fixing avatar spacing and shape

### PR DESCRIPTION
**Pull request by KoushikAD1234**
Fixes #132 (Fixing up Avatar spacing and shape )

### CODE REVIEW

File: src/components/ChatHeader/ChatHeader.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ const ChatHeader = ({
       </Box>
       <Box display="flex" alignItems="center">
         {avatarUrl && (
-          <img width="20px" height="20px" src={avatarUrl} alt="avatar" />
+          <img width="20px" height="20px" src={avatarUrl} alt="avatar" style={{marginRight:"15px", borderRadius:"10px"}} />
```
Comment (sidmohanty11) [inline src/components/ChatHeader/ChatHeader.js:214]: Have you checked for all screen sizes? Also, can you paste screenshots of UI for each screen. Please try these for a bigger channel name too.

**Review by sidmohanty11 — CHANGES_REQUESTED** [review]
Thanks @KoushikAD1234 for the PR!

File: src/components/ChatHeader/ChatHeader.js (hunk 1 above)
Comment (KoushikAD1234) [reply src/components/ChatHeader/ChatHeader.js:214]: Hey @sidmohanty11 I am attaching the screenshot of the changes and it is fine for every screen size. Please review it once. And also I found a BUG in ChatHeader component that is when we use larger channel name and try to minimize the screen the ChatHeader component will overflow the screen, I am planning to make a seperate PR for this issue.
Screenshot from 2023-02-18 19-08-51

Screenshot from 2023-02-18 19-09-31

**Comment by KoushikAD1234** [comment]
Hey @sidmohanty11 I am attaching the screenshot of the changes and it is fine for every screen size. Please review it once
Screenshot from 2023-02-20 18-57-58
Screenshot from 2023-02-18 19-08-51
Screenshot from 2023-02-18 19-09-31

**Review by sidmohanty11 — CHANGES_REQUESTED** [review]
Thanks @KoushikAD1234 ! Looks good, lets just remove the shape as of now because RC uses the default square shape for everything - and change the `img` to `Avatar` component from fuselage.

Ref: https://rocketchat.github.io/fuselage/fuselage/master/?path=/story/data-display-avatar--default

**Pull request closed without merging**